functions/Find-DbaOrphanedFile.ps1

```diff
@@ -0,0 +1,225 @@
+﻿
+Function Find-DbaOrphanedFile
+{
+<#
+.SYNOPSIS 
+Find-DbaOrphanedFile moves/removes orphaned database files

```

Comment:
I don't see any code that shows you do any "moves/removes" for orphaned files found. Should be reworded.